### PR TITLE
[IMP] GH #92 Help attrs for US IDs

### LINKED ISSUE #92: Unclear mandatory fields for creating doctor & pharmacy
When I was creating a new doctor & new pharmacy, it said that I had missed mandatory fields, but I couldn't tell which we were mandatory. Also, depending on the user, it might be helpful to have more guidance about what specific required ID numbers look like (number/type of characters, etc).

### PR DESCRIPTION
**Pull request by kellylougheed**
Adds more helper text about valid input to ID fields.

Fixes #92

### CODE REVIEW

**Comment by lasley** [comment]
Looks like lint is failing - https://travis-ci.org/LasLabs/vertical-medical/jobs/191725033#L319

Ignore all the tabs instead of spaces ones (those won't actually fail your build anyways). I'll actually submit something to fix that all right now.

File: medical_insurance_us/models/medical_insurance_template.py [hunk 1]
```diff
@@ -9,10 +9,10 @@ class MedicalInsuranceTemplate(models.Model):
     _inherit = 'medical.insurance.template'
     group_number = fields.Char()
     rx_bin = fields.Char(
-        help='RX BIN Number',
+        help='RX BIN Number - six-digit number',
```
Comment (lasley) [inline medical_insurance_us/models/medical_insurance_template.py]: We could also add `size` attributes here if we know the sizes

File: medical_insurance_us/models/medical_insurance_template.py [hunk 2]
```diff
@@ -9,10 +9,10 @@ class MedicalInsuranceTemplate(models.Model):
...
     )
     rx_pcn = fields.Char(
-        help='RX Processor Control Number (PCN)',
+        help='RX Processor Control Number (PCN) - up to 10 chars',
```
Comment (lasley) [inline medical_insurance_us/models/medical_insurance_template.py]: Switching between numeric and alpha representations of the numbers. I'm indifferent, but pick one and stick with it 😉

File: medical_medicament_us/models/medical_medicament_ndc.py [hunk 3]
```diff
@@ -11,7 +11,7 @@ class MedicalMedicamentNdc(models.Model):
 
     name = fields.Char(
         string='NDC',
-        help='National Drug Code',
+        help='National Drug Code - 10-digit number',
```
Comment (lasley) [inline medical_medicament_us/models/medical_medicament_ndc.py]: NDC can be 10 or 11 actually - http://phpa.dhmh.maryland.gov/OIDEOR/IMMUN/Shared%20Documents/Handout%203%20-%20NDC%20conversion%20to%2011%20digits.pdf

File: medical_patient_us/models/medical_patient.py [hunk 4]
```diff
@@ -28,7 +28,7 @@ class MedicalPatient(models.Model):
         inverse=lambda s: s._inverse_identification(
             'drivers_license_us', 'DL-US',
         ),
-        help='US Driver\s License',
+        help='US Driver\s License - varies by state',
```
Comment (lasley) [inline medical_patient_us/models/medical_patient.py]: missing an apostrophe after the `\`-  Driver\'s

File: medical_pharmacy_us/models/medical_pharmacy.py [hunk 5]
```diff
@@ -17,7 +17,7 @@ class MedicalPharmacy(models.Model):
         inverse=lambda s: s._inverse_identification(
             'napb_num', 'NAPB',
         ),
-        help='National Boards of Pharmacy Id #',
+        help='National Boards of Pharmacy Id # - seven-digit number',
```
Comment (lasley) [inline medical_pharmacy_us/models/medical_pharmacy.py]: Switching between hyphenated and non with `number-digit` vs `number digits`. Again indifferent, but keep it consistent.

File: medical_pharmacy_us/models/medical_pharmacy.py [hunk 6]
```diff
@@ -28,7 +28,7 @@ class MedicalPharmacy(models.Model):
         inverse=lambda s: s._inverse_identification(
             'medicaid_num', 'MEDICAID',
         ),
-        help='Medicaid Id #',
+        help='Medicaid Id # - nine digits and one alpha character',
```
Comment (lasley) [inline medical_pharmacy_us/models/medical_pharmacy.py]: s/Id/ID

File: medical_physician_us/models/medical_physician.py [hunk 7]
```diff
@@ -28,7 +28,8 @@ class MedicalPhysician(models.Model):
         inverse=lambda s: s._inverse_identification(
             'dea_num', 'DEA',
         ),
-        help='Drug Enforcement Agency #',
+        help='''Drug Enforcement Agency # - \
```
Comment (lasley) [inline medical_physician_us/models/medical_physician.py]: Best to break this string off. Current strategy is adding a bunch of blank whitespace.

```
help='Drug Enforcement Agency # - '
     'two alpha characters and seven digits'
```

**Review by lasley — CHANGES_REQUESTED** [review]

**Comment by lasley** [comment]
@kellylougheed - is this ready for review?

**Comment by kellylougheed** [comment]
@lasley yes!

**Comment by lasley** [comment]
@kellylougheed - sweet -so in our repos when something is ready after changes have been made, drop a comment saying so & update the tags. In repos you don't have write access to, the comment should also contain a note for maintainers to update the tag.

**Review by lasley — APPROVED** [review]

**Pull request merged**